src/Microsoft.OData.Core/UriParser/Aggregation/AggregateExpression.cs

```diff
@@ -8,43 +8,63 @@ namespace Microsoft.OData.UriParser.Aggregation
 {
     using Microsoft.OData.Edm;
     using Microsoft.OData.UriParser;
+    using System;
+    using System.Collections.Generic;
 
     /// <summary>
     /// A aggregate expression representing a aggregation transformation.
     /// </summary>
-    public sealed class AggregateExpression
+    public abstract class AggregateExpression

```

Comment:
I'm not sure I'm getting this comment. The only constructor of the class is protected. Isn't the default constructor present only in the absence of other constructors?